widgets/LiquidContainer/LiquidContainer.hpp

```diff
@@ -0,0 +1,35 @@
+#ifndef LIQUID_CONTAINER_HPP
+#define LIQUID_CONTAINER_HPP
+
+#include <mvp/View.hpp>
+#include <gui/template_screen/TemplatePresenter.hpp>
+#include <touchgfx/widgets/Image.hpp>
+#include <BitmapDatabase.hpp>
+#include <touchgfx/containers/Container.hpp>
+#include <touchgfx/widgets/TextAreaWithWildcard.hpp>
+#include <texts/TextKeysAndLanguages.hpp>
+
+using namespace touchgfx;
+
+class LiquidContainer : public Container
+{
+public:
+  LiquidContainer();
+  void init(int containerImageId, int liquidImageId, 
+	    int containerTopOffset, int containerBottomOffset, 
+	    int liquidSpeed, 
```

Comment:
Names of members and methods should be modified in accordance with the new name of the widget.